app/views/foods/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= link_to 'Add new food', new_food_path, :class => 'button float-right margin-bottom-bttn' %>
+<div class="clear margin-table">
+  <table class="table">
+    <thead class="thead-dark">
+      <tr>
+        <th>Food</th>
+        <th>Measurement unit</th>
+        <th>Unit price</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @foods.each do |food| %>
+        <tr>
+        <td><%= food.name%></td>
+        <td><%= food.measurementUnit%></td>
+        <td><%= food.price%></td>
+        <td><%= button_to "Delete", food, method: :delete, class: 'btn btn-primary' %></td>
```

Comment:
- [ OPTIONAL] Great job Team 👏🏻  Just make sure the index page for food list matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/food_index.png?raw=true). I will be better to add unities: `grams` for the Measurement unit and `$` for the Unit price 😉 👍🏻 

- Your Design:

-  ![food](https://user-images.githubusercontent.com/79058364/175202636-abb95e09-e9fa-4fe0-aa70-4180b5fbada7.png)

- Original Design:

![foodgod](https://user-images.githubusercontent.com/79058364/175202886-6bfc8c0a-be7a-48e1-b285-82e4944ad9a0.png)